File: C_Sharp/p1768.cs
public class Solution {
    public string MergeAlternately(string word1, string word2) {
        int length1 = word1.Length;
        int length2 = word2.Length;
        int maxL = Math.Max(length2, length1);
        string result = "";
        for(int index = 0; index < maxL; index++){
            if(index < length1){
                result += word1[index];
            }
            if(index < length2){
                result += word2[index];
            }
        }
        return result;
    }
}

File: C_Plus_Plus/p1768.cpp
#include <string>
#include <algorithm>
using namespace std;

class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int length1 = word1.length();
        int length2 = word2.length();
        int maxLength = max(length1, length2);
        string result = "";
        for(int index = 0; index < maxLength; index++){
            if(index < length1){
                result.push_back(word1[index]);
            }
            if(index < length2){
                result.push_back(word2[index]);
            }
        }
        return result;
    }
};

int main(){

}